packages/cli/src/cmds/beacon/options.ts

```diff
@@ -102,6 +102,13 @@ export const beaconPathsOptions: ICliCommandOptions<IBeaconPaths> = {
     type: "string",
   },
 
+  archiveDir: {
```

Comment:
This name is way to vague, only debuggers should use this. The name should imply it's for debugging
```
persistInvalidSszObjectsDir
```